README.adoc

```diff
@@ -27,52 +27,48 @@ NOTE: This repository contains the guide documentation source. To view the guide
 
 Explore how to enable and customize tracing of JAX-RS and non-JAX-RS methods by using MicroProfile OpenTracing and the Zipkin tracing system.
 
-
 :inv-url: http://localhost:9081/inventory/systems
 :inv-url-docker: http://localhost:9080/inventory/systems
 :sys-url: http://localhost:9080/system/properties
 :zipkin-url: http://localhost:9411
 
-
 == What you'll learn
 
-You will learn how to enable automatic tracing for JAX-RS methods as well as create custom tracers
+You'll learn how to enable automatic tracing for JAX-RS methods as well as create custom tracers
```

Comment:
You'll learn how to enable automatic tracing for JAX-RS methods as well as create custom tracers
--->
You'll learn how to enable automatic tracing for JAX-RS methods and how to create custom tracers.